.ci/build-master.yml

```diff
@@ -38,11 +34,10 @@ phases:
           PathtoPublish: '_platformrelease'
           ArtifactName: macOS
 
-  - phase: Phase_3
-    displayName: Windows
-    condition: succeeded()
-    queue:
-      name: Hosted VS2017
+  - job: Windows
+    timeoutInMinutes: 120
```

Comment:
The change to use `jobs` allowed to temporarily increase the available time for Windows